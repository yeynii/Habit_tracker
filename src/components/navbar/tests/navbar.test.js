import React from "react";
import { render, screen } from "@testing-library/react";
import Navbar from "../navbar";
import renderer from "react-test-renderer";
import "@testing-library/jest-dom";

describe('Navbar', () => {
  const habits = [
      { id: 1, name: "Reading", count: 1 },
      { id: 2, name: "Running", count: 0 }
    ];

  it('renders', () => {
    const component = renderer.create(<Navbar totalCount={4}/>)
    expect(component.toJSON()).toMatchSnapshot();
  });
})