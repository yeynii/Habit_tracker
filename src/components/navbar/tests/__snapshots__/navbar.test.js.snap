// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar renders 1`] = `
<header>
  <i
    className="fas fa-leaf"
  />
  <span>
    Habit Tracker
  </span>
  <span
    className="active-count"
    data-testid="total-count"
  >
    4
  </span>
</header>
`;
